package.json

```diff
@@ -37,13 +37,16 @@
         "lint:fix:md": "markdownlint --fix  \"**/*.md\"",
         "lint:scss": "stylelint \"**/*.scss\"",
         "lint:fix:scss": "stylelint \"**/*.scss\" --fix",
+        "format": "prettier --write \"**/*.+(js|json|scss)\"",
+        "format:check": "prettier --check \"**/*.+(js|json|scss)\"",
```

Comment:
Can we name these scripts `fmt` and `fmt:check`, for consistency with the eslint/rewrite repo? Also, can you add running `fmt:check` to [`ci.yml`](https://github.com/eslint/eslint.org/blob/main/.github/workflows/ci.yml) to ensure that the codebase is always properly formatted? Example in the eslint/rewrite repo:

https://github.com/eslint/rewrite/blob/66a97f9ebb09ec29a2b12c05fbd215b18363f755/.github/workflows/ci.yml#L30-L31